### ISSUE TITLE
Likelihood validation when using minibatch

### ISSUE THREAD

**Issue by EricR86** [hoffmangroup/segway#99]
**Original report ([BitBucket issue](https://bitbucket.org/hoffmanlab/segway/issues/99)) by Rachel Chan (Bitbucket: [rcwchan](https://bitbucket.org/%7B20430f8d-4e6b-48cc-a8d8-c64868bf7e79%7D/)).**

----------------------------------------
Currently, Segway's winning set of params is chosen based on the instance with the best last likelihood. Due to minibatch's highly variational nature, this may not be the best way to choose the winner.

Instead, should minibatch automatically test likelihood on a validation set and choose based on whichever set of params has the best validation test likelihood (even if it's not the last one)?

If yes, should the region be automatically inferred or should it be required to be specified using something such as --validation-coords?

If automatically inferred, what size should the validation set be? I know we previously said '10x the training set', but what if the minibatch fraction specified is extremely large (ie, 0.10 or greater)?
